example/src/main/java/com/quickbirdstudios/example/ui/main/MainActivity.kt

```diff
@@ -201,6 +201,16 @@ class MainActivity : AppCompatActivity() {
             }
         }
 
+        surveyView.onStepBegin = { step ->
+            when (step) {
+                is QuestionStep -> {
+                    Log.w ("ASDF", "step '${step?.id}': ${step.title} (${step.text})")
+                }
+                else ->
+                    Log.w ("ASDF", "step '${step?.id}'")
```

Comment:
I would show a Toast or something in the `View` just to make sure view interactions do not break. But logs are fine as well 👍